runFtest.py

```diff
@@ -33,9 +33,10 @@ def buildcards(odir, v1n, v2n, options):
     if odir == "":
         odir = os.path.dirname(ifile)
         print "using default output dir:", odir
-    create_cards = "python create_datacard.py --inputfile=%s --carddir=%s --nbins=%i --nDataTF=%i --passBinName=%s" % (ifile, odir, options.n, v1n, options.passBinName)
+    create_cards = "python create_datacard.py --inputfile=%s --carddir=%s --nbins=%i --nDataTF=%i --passBinName=%s --blinded %s" % (ifile, odir, options.n, v1n, options.passBinName, options.blinded)
     if options.testMCTF:
-        create_cards = "python create_datacard.py --inputfile=%s --carddir=%s --nbins=%i --nMCTF=%i --passBinName=%s" % (ifile, odir, options.n, v1n, options.passBinName)
+        create_cards = "python create_datacard.py --inputfile=%s --carddir=%s --nbins=%i --nMCTF=%i --passBinName=%s --blinded %s" % (
+            ifile, odir, options.n, v1n, options.passBinName, options.blinded)
 
     combineCards = "cd %s/HHModel; combineCards.py pass=pass%s.txt fail=fail.txt > HHModel_combined.txt; text2workspace.py HHModel_combined.txt ;cd -" % (odir, options.passBinName)
```

Comment:
Hi @irenedutta23, I noticed that for unblinded case, this line 41 should be changed to

combineCards = "cd %s/HHModel; combineCards.py pass=SR%s.txt fail=fitfail.txt > HHModel_combined.txt; text2workspace.py HHModel_combined.txt ;cd -" % (odir, options.passBinName)

Reply:
Thanks for the pointer, this is done now.